pkgs/development/python-modules/keyutils/default.nix

```
@@ -1,4 +1,4 @@
-{ lib, buildPythonPackage, fetchFromGitHub, keyutils, pytest-runner, pytest }:
+{ lib, buildPythonPackage, cython, fetchFromGitHub, keyutils, pytest-runner, pytest }:
```

```suggestion
{ lib
, buildPythonPackage
, cython
, fetchFromGitHub
, keyutils
, pytestCheckHook
}:
```

Unfold the arguments, so they're one per line. This makes diffs easier to grok and results in fewer merge conflicts.